eva-lib/src/main/java/uk/ac/ebi/variation/eva/lib/storage/metadata/VariantSourceEvaproDBAdaptor.java

```diff
@@ -142,12 +142,12 @@ public QueryResult getSourceDownloadUrlByName(String filename) {
             rs = pstmt.executeQuery();
             
             if (rs.next()) {
-                URL url = new URL("ftp:/" + rs.getString(1));
+                URL url = new URL("ftp:/" + rs.getString(2));
                 long end = System.currentTimeMillis();
-                qr = new QueryResult(null, ((Long) (end - start)).intValue(), 1, 1, null, null, Arrays.asList(url));
+                qr = new QueryResult(rs.getString(1), ((Long) (end - start)).intValue(), 1, 1, null, null, Arrays.asList(url));
             } else {
                 long end = System.currentTimeMillis();
-                qr = new QueryResult(null, ((Long) (end - start)).intValue(), 0, 0, null, null, new ArrayList<>());
+                qr = new QueryResult(rs.getString(1), ((Long) (end - start)).intValue(), 0, 0, null, null, new ArrayList<>());
```

Comment:
I this will fail, if rs.next() returns false, rs.getString() could launch an SQLException